### PR TITLE
Mapfish print - Client-side

### PR DESCRIPTION
**Pull request by kalbermattenm**
This implements the client side for MapFish print.

@voisardf, please review.

I think that you should from now on also develop on this branch. Otherwise you are going to lose track of what I am doing. Maybe we should discuss that on Tuesday.

This PR is meant to be merged into the mapfish_print banch

### CODE REVIEW

File: crdppf/views/printproxy.py [hunk 1]
```diff
@@ -92,7 +94,7 @@ def report_create(self):
         # Specify correct content type in headers
         h = dict(self.request.headers)
         h["Content-Type"] = "application/json"
-
+        print _string
```
Comment (kalbermattenm) [inline crdppf/views/printproxy.py]: Should be removed...

File: crdppf/static/js/Crdppf/createReport.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        if (status === true) {
+            clearInterval(this.interval);
+            this.getReport(ref);
+        }
+    },
+
+    getStatus: function(ref) {
+
+        Ext.Ajax.request({
+            url: Crdppf.printReportStatusUrl + ref + '.json',
```
Comment (kalbermattenm) [inline crdppf/static/js/Crdppf/createReport.js]: No spaces

**Comment by kalbermattenm** [comment]
There is bug: try to print 2 extracts...

File: crdppf/static/js/Crdppf/main.js [hunk 3]
```diff
@@ -291,29 +292,20 @@ Crdppf.init_main = function(lang) {
...
+                            "/",
+                            Crdppf.currentProperty.attributes.idemai
+                        ].join('');
+
+                        this.Report.create(url, pdfMask);
+
+                    },
+                    scope: this
                 }
             },{
```
Comment (voisardf) [inline crdppf/static/js/Crdppf/main.js]: Are the spaces and intendations ok here?

**Comment by voisardf** [comment]
just minor comments. LGTM otherwise.

File: crdppf/static/js/Crdppf/main.js (hunk 3 above)
Comment (kalbermattenm) [inline crdppf/static/js/Crdppf/main.js]: Hmmm.... I like it... for readability...

Do you want me to remove them ?

**Pull request merged**